src/stylesheets/Navbar.css

```diff
@@ -1,10 +1,14 @@
 .navbar {
   width: 100%;
   height: 150px;
+  background-color: #fff;
 }
 
 .navbar nav {
   display: flex;
+  position: relative;
+
+  /* justify-content: space-around; */
```

Comment:
- [ ] Kindly remove the commented code to achieve a cleaner codebase.
- [ ] Please make your header i.e the name of your bookstore and the profile icon be contained within the book's card
- [ ] Kindly reduce the size of the progress bar to match the Zeplin design

<img width="551" alt="ApplicationFrameHost_wDlNn9HL07" src="https://user-images.githubusercontent.com/81468675/162135870-da1c7dbe-729e-4d1f-887e-fe25eb1e8f50.png">

Reply:
Thank you @julie-ify  for this review. I have made the requested changes